synapse/swarm/opers/common.py

```diff
@@ -23,6 +27,34 @@ def runSyntaxCheck(self):
         pass
 
     def run(self):
+        debug_count = self.query.opt('debug:count')
+        debug_size = self.query.opt('debug:time')
+        debug_time = self.query.opt('debug:size')
+
+        if debug_count:
+            count = len(self.query.data())
+
+        if debug_size:
+            size_bytes = len(msgenpack(self.query.data()))
+
+        if debug_time:
+            start = time.time()
+
+        rslt = self._run()
+
+        if debug_time:
+            duration_ms = int((time.time() - start) * 1000)
+            self.query.setInstDebug('time', duration_ms)
+
+        if debug_size:
```

Comment:
if we care about the size of data being serialized inbound from the cortexes, we may have to enable some additional recording in getTufosByPropFrom().  I'm not sure that the msgpack size of the result set at each oper is a very good metric for anything.